Core/EventHandlers.lua

```diff
@@ -835,14 +835,21 @@ end
 
 local tooltipLeftText1 = _G["GameTooltipTextLeft1"]
 
+function StripColourCode(input)
```

Comment:
This pollutes the global namespace, so it'd be better to make it ``local`` or put it in another table somewhere. Since it isn't used anywhere else and we haven't any text-processing modules, I don't know a fitting place to move it to. I guess it'll just have to be refactored out later if it is to be reused.

Also, please use the US spelling instead of the UK version :) While I'm European myself, we should likely be consistent and use the same spelling that's already present in the code, by virtue of the original author being American.

Reply:
Thanks for the input. Changes made.